EU4toV2/Source/V2World/Country/ModCommons.cpp

```diff
@@ -0,0 +1,69 @@
+#include "ModCommons.h"
+#include "CommonRegexes.h"
+#include "Configuration.h"
+#include "OSCompatibilityLayer.h"
+#include "ParserHelpers.h"
+
+V2::ModCommons::ModCommons(const std::string& filename)
+{
+	registerKeyword("color", [this](const std::string& unused, std::istream& theStream) {
+		const commonItems::stringOfItem colorStr(theStream);
+		color = colorStr.getString();
+	});
+	registerKeyword("graphical_culture", [this](const std::string& unused, std::istream& theStream) {
+		const commonItems::singleString graphicalCultureStr(theStream);
+		graphicalCulture = graphicalCultureStr.getString();
+	});
+	registerKeyword("party", [this](const std::string& unused, std::istream& theStream) {
+		const mappers::PartyType newPartyType(theStream);
+		const Party newParty(newPartyType);
+		parties.push_back(newParty);
+	});
+	registerKeyword("unit_names", [this](const std::string& unused, std::istream& theStream) {
+		const commonItems::stringOfItem unitNamesStr(theStream);
+		unitNames = unitNamesStr.getString();
+	});
+	registerRegex(commonItems::catchallRegex, commonItems::ignoreItem);
+
+	const auto& file = determineFilePath(filename);
+	if (file)
```

Comment:
if (commonItems::DoesFileExist(theConfiguration.getVic2Path()+ "/common/countries/" + filename))
{ ... }
no need for separate function.